g2gmobile/app.json

```diff
@@ -8,9 +8,9 @@
     "version": "1.0.0",
     "orientation": "portrait",
     "primaryColor": "#cccccc",
-    "icon": "./assets/icons/app.png",
+    "icon": "./assets/icons/GTG-Box-App.png",
```

Comment:
@damianhouse Can you include this file with this PR? This asset is not in this repo